sherpa/optmethods/src/minim.hh

```diff
@@ -29,26 +29,41 @@ namespace sherpa {
       return ifault;
     }
 
-
     void minim( std::vector<real>& P, const std::vector<real>& STEP,
                 int NOP, real& FUNC, int MAXNFEV, int IPRINT, real STOPCR,
                 int IQUAD, real SIMP, std::vector<real>& VC, int& IFAULT,
                 int& NEVAL, const sherpa::Bounds<real>& LIMITS ) {
-      return MINIM( P, STEP, NOP, FUNC, MAXNFEV, IPRINT, STOPCR, IQUAD, SIMP,
-                    VC, IFAULT, NEVAL, LIMITS);
+      MINIM( P, STEP, NOP, FUNC, MAXNFEV, IPRINT, STOPCR, IQUAD,
+             SIMP, VC, IFAULT, NEVAL, LIMITS );
+      return;
     }
 
   private:
 
     Func usr_func;
     Data usr_data;
 
+    void reflect_about_boundary( int npar, std::vector<real>& par,
```

Comment:
This is not a lien on this PR, but I assume this is not a general-purpose operation - i.e. it should only be used here, and not everywhere else we have code that calls `sherpa::Opt<Data, real>::are_pars_outside_limits`?